shims/shims_for_IE.js

```diff
@@ -18,9 +18,9 @@ require('html5-history-api');  // HTML5 History: IE9.
 if (!Object.hasOwnProperty('name')) {
   Object.defineProperty(Function.prototype, 'name', {
     get: function getName() {
-      const matches = this.toString()
+      var matches = this.toString()
```

Comment:
Yeah webpack just doesn't have anything setup to transpile .js files, so the `const` keywords were not going changed. I've changed it now to use a .ts extension and replaced the --ignore-path arg.